### DISCUSSION TITLE
Nodes do not fold any more

### DISCUSSION THREAD

**Discussion by doctomlug** [freeplane/freeplane#1362] [Help] [answered]
Hi all,
I use mind maps to work with contracts and end up with very large maps including many text nodes. Formatting is usually done by styles and almost never edited directly. I use version 1.11.2 (company computer unable to update often)

While editing I discovered nodes that will not fold any more. I tried cut/past and exporting into a new map to "reset" the nodes but ended up unsuccessful. Exporting the map to clickable html yields empty check boxes without functionality for the corresponding nodes.

I was able to "copy" the problem and reduce it into a simple demo map (attached zip file). 

Since I discovered the problem in a map I use and modify almost daily and I am unaware when the issue has been introduced, I do not know what script, attribute change or other action from my side introduced the issue - I am kind of lost.

It appears to me that the internal structure of the node is corrupted and I am looking for a practical way to restore the node / solve the issue without having to copy / paste every single node of the map.

[NoFolding.zip](https://github.com/freeplane/freeplane/files/12437136/NoFolding.zip)

Thanks for your help

Edit: Updated to 1.11.6. No changes

**Comment by euu2021** [comment, marked as answer]
This: `Navigate->Fold->Always unfolded node (set/reset)`

**Reply by doctomlug** [reply]
This did it. Thank you
